src/Drawing.ts

```diff
@@ -586,15 +586,15 @@ const renderShape: (shape: Shape) => (ctx: CanvasRenderingContext2D) => IO.IO<Ca
       return (ctx) =>
         pipe(
           readOnlyArrayTraverseIO(shape.shapes, (s) => pipe(ctx, renderShape(s))),
-          IO.chain(() => IO.of(ctx))
+          IO.chain(() => () => ctx)
```

Comment:
this is

```ts
IO.map(() => ctx)
```

Reply:
See https://github.com/gcanti/graphics-ts/pull/7/commits/d3848bcfd2b47782484a1da84fe31d5e2cae11f8